SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -993,25 +1003,12 @@ void USpatialActorChannel::InitializeHandoverShadowData(TArray<uint8>& ShadowDat
 {
 	const FClassInfo& ClassInfo = NetDriver->ClassInfoManager->GetOrCreateClassInfoByClass(Object->GetClass());
 
-	uint32 Size = 0;
+	ShadowData.AddZeroed(ClassInfo.HandoverPropertiesSize);
```

Comment:
Is there a chance this can be called twice? Potentially appending bytes instead of replacing? Maybe if an actor channel is recycled!?
```suggestion
	ShadowData.SetNum(ClassInfo.HandoverPropertiesSize);
	FMemory::Memset(ShadowData.GetData(), 0x00, ClassInfo.HandoverPropertiesSize);
```

Reply:
Usage suggests this can't happen - `InitializeHandoverShadowData` is called with `MakeShared<TArray<uint8>>().Get()` so these are empty. Agreed on the `SetNum` instead of `AddZeroed` though as this would better reflect the method's name. Not sure about `Memset` as we always do `FProperty::InitializeValue` on the bytes.